tests/e2e/instrumentation-java/01-install-app.yaml

```diff
@@ -15,6 +15,10 @@ spec:
         sidecar.opentelemetry.io/inject: "true"
         instrumentation.opentelemetry.io/inject-java: "true"
     spec:
+      securityContext:
```

Comment:
Would this work on OpenShift? My understanding is that it needs a special SCC configuration done by the cluster-admin in order to allow this config at the workload level.

Note also that OpenShift (and likely other platforms) sets a random user ID to the actual processes, so, if you need the user ID to be a specific number, your code will certainly break at runtime.

Reply:
I don't know if this works on OCP or other platforms. 

The previous commit shows that it fails if the image does not have `go+r`.